requirements.txt

```diff
@@ -4,10 +4,12 @@ Flask-API==1.1
 Flask-SQLAlchemy==2.4.1
 psycopg2-binary==2.8.3
 pymysql==0.9.3
+git-pylint-commit-hook
```

Comment:
Sorry, I just noticed. I think we can remove this line as well.

Reply:
this is because we have it in the Testing area.

Reply:
Do we? Where is it?

Reply:
Down in the requirements.txt only where you've added it. (My phrasing might be wrong. I didn't mean testing code. Only testing section of requirements.txt)

Reply:
Got it.